backend/src/Request/Admin/AdminProfile/AdminProfileUpdateBySuperAdminRequest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Request\Admin\AdminProfile;
+
+use App\Entity\ImageEntity;
+
+class AdminProfileUpdateBySuperAdminRequest
+{
+    private int $id;
+
+    private string $name;
+
+    /**
+     * @var string|null
+     */
+    private $phone;
+
+    /**
+     * @var string|null
+     */
+    private $imagePath;
+
+    /**
+     * @var ImageEntity|null
+     */
+    private $image;
```

Comment:
must be images

Reply:
It's name similar to image field in the AdminProfileEntity, why we have to name it "images"?